src/tools/bin/tbg

```diff
@@ -423,8 +427,12 @@ fi
 start_dir=`dirname $0`
 
 if [ -d "$job_outDir" ] ; then
-    echo "job name already in use, can't create new folder"
-    exit 1
+	if [ "$force" = "true" ] ; then
+		echo "Warning: using existing folder for job on user-request ( -f )"
+	else
+		echo "job name already in use, can't create new folder"
```

Comment:
the old error message was not well put, maybe write `destinationPath already in use, ...`